docs/developers/contributing.md

```diff
@@ -223,17 +257,9 @@ git push -u origin your-branch-name
 You can then make a
 [pull-request](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#making-a-pull-request) to `napari`'s `main` branch.
 
-## Building the docs
-
-From the project root:
-```sh
-make docs
-```
-
-The docs will be built at `docs/_build/html`.
+## Contributing documentation
```

Comment:
I would actually move this section up to just underneath setting up a `dev` installation. Pretty much everything else on this page is less important and less frequently used imo.

Reply:
Moved, thanks!